public/app/features/trails/DataTrail.test.tsx

```diff
@@ -276,12 +264,6 @@ describe('DataTrail', () => {
               getFilterVar().setState({ filters: [{ key: 'zone', operator: '=', value: 'b' }] });
             });
 
-            it('should sync state with url', () => {
-              expect(decodeURIComponent(locationService.getSearchObject()['var-filters']?.toString()!)).toBe(
-                'zone|=|b'
-              );
-            });
```

Comment:
Is it no longer possible to test for URI changes (or is it considered redundant from this context)?

Reply:
You can easily test url changes , just need to render the scene component and wrap it in UrlSyncContextProviser

Reply:
Reason I removed these as we have unit tests for variable url sync in the scenes lib so not really needed here